straxen/url_config.py

```diff
@@ -229,3 +230,8 @@ def format_arg(arg: str, **kwargs):
     '''apply pythons builtin format function to a string
     '''
     return arg.format(**kwargs)
+
+
+@URLConfig.register('interpolatingmap')
+def load_map(some_map, method='WeightedNearestNeighbors', **kwargs):
```

Comment:
I guess the `method` is the most important argument. But if analysts actually do use the kwargs options in the InterpolatingMap then its an easy fix to add a cleaning step of the kwargs before they are passed to the various interpolating methods inside the __init__ method of InterpolatingMap. I have an implementation of this in the URLConfig class `URLConfig.filter_kwargs(func, kwargs)`, its a static method so can be called from the class and it will return the subset of kwargs accepted by func.

Reply:
I did see this filter_kwargs method but I naively assumed it was always being used. Sorry I still don't fully understand though -- what do I need to change to filter the kwargs? I agree this would be good to have

Reply:
the filter_kwargs is applied for the protocol functions, but if the protocol accepts kwargs then all of them will be passed in. What I meant is that you can use the function to filter the kwargs that are passed on by the `__init__` method of the InterpolatingMap class. Currently it just passes along all kwargs to the various interpolating methods. I can edit the class and push it later if you think this would be useful, not sure if these extra interpolating options are ever used

Reply:
I just made a PR #823 for this just in case, you never know when you may need to change some parameter of the interpolation method...